cmd/karavictl/cmd/role_update.go

```diff
@@ -30,8 +30,8 @@ import (
 func NewRoleUpdateCmd() *cobra.Command {
 	roleUpdateCmd := &cobra.Command{
 		Use:   "update",
-		Short: "Update one or more Karavi roles",
-		Long:  `Updates one or more Karavi roles`,
+		Short: "Update the quota of one or more Karavi roles",
+		Long:  `Updates the quota of one or more Karavi roles`,
```

Comment:
Should we still refer to this as a "Karavi" role?

Reply:
No, updated.